### DISCUSSION TITLE
Are there any plans to support deployment in mainland China in the future?

### DISCUSSION THREAD

**Discussion by maxwezou** [supabase/supabase#27678] [General]
Supabase is a great product and I can't wait to use it as a backend on all my projects. Most of the users my project serves are in China, but requests from mainland China are unstable due to GFW. It would be great if I could create a new Supabase project to select the China region.

**Comment by ShaysFrame** [comment]
Seems like no hope

**Comment by xunjianxiang** [comment]
With such a large market, we can choose to cooperate with Alibaba Cloud or Tencent Cloud in China

**Comment by sidChouaib** [comment]
Is there any updates regarding Supabase support for china ? surprisingly a person has tested the app there for me for few minutes and it worked just fine without a vpn.
The server is in EU West.
